app/models/timer.rb

```diff
@@ -0,0 +1,48 @@
+class Timer < ActiveRecord::Base
+
+  belongs_to :player
+
+  # ALTERNATIVE start and stop methods
+  
+  # def start_time
+  #   @running = true
+  #   @start_time = Time.now
+
+  #   limit if @time_limit
+
+  #   self
+  # end
+
+  # def stop_time
+  #   if @running
+  #     defuse
+  #     @end_time = Time.now
+  #     @running = false
+  #     @total_time += (@end_time - @start_time)
+  #   else
+  #     nil
+  #   end
+  # end
+
+
+  def start_time
+    Timer.create(player_id: current_player.id, time: Time.now)
```

Comment:
Yes, need to rework this because that isn't the intent (I think) - even though each new game will use new timers, but not each start of the timer because a player in a game can hit the start/stop several times during a game.